classes/services/NavigationMenuService.inc.php

```diff
@@ -92,8 +93,20 @@ function getDisplayStatusCallback($hookName, $args) {
 				$navigationMenuItem->setIsDisplayed($context && $context->getSetting('publishingMode') != PUBLISHING_MODE_NONE);
 				break;
 			case NMI_TYPE_SUBSCRIPTIONS:
+				if ($context) {
+					$paymentManager = \Application::getPaymentManager($context);
+					$navigationMenuItem->setIsDisplayed($context->getSetting('paymentsEnabled') && $paymentManager->isConfigured());
+				} else {
+					$navigationMenuItem->setIsDisplayed(false);
```

Comment:
Is this `false` by default? Do we need to set it in this case?

Reply:
@NateWr yes you are right. Fixed